docs/spec/sanitation.md

```diff
@@ -0,0 +1,22 @@
+_Author_: @KamalikaSene \
+_Created_: 2024/06/25 \
+_Edition_: Swan Lake  
+
+# Sanitation for OpenAPI specification
+
+This records the sanitization done when converting OAS from version 3.1.0 to 3.0.0 using APIMatic's conversion tool. These sanitation's are done to reduce the errors which occur when generating the client.
```

Comment:
This should come as the 1st entry under sanitations

Reply:
Ack, all relevant changes made.